components/com_finder/src/Model/SuggestionsModel.php

```diff
@@ -108,8 +109,8 @@ protected function getListQuery()
             ->where('l.access IN (' . implode(',', $groups) . ')')
             ->where('l.state = 1')
             ->where('l.published = 1');
-
-        return $termQuery;
+        $aQuery->select('DISTINCT(o.term) FROM (' . $termQuery . ') AS o');
```

Comment:
```suggestion
        $aQuery->select('DISTINCT o.term FROM (' . $termQuery . ') AS o');
```
Even if it works we should not use the brackets syntax as that is not officially supported according to MySQL and MariaDB docs.

Reply:
ok 
thanks